### PR TITLE
Feature/create profile page

### PR DESCRIPTION
**Pull request by clarafitzgerald**
Added Create Profile page, linked in to firebase & added logic to direct user to matches/create profile dependent on whether they're registered

### CODE REVIEW

File: src/pages/CreateProfile/BasicInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
+import React from "react";
```
Comment (mayakeeley) [inline src/pages/CreateProfile/BasicInfo.tsx:1]: Should: resolve errors in console
Getting the following error in the console from this page:

Warning: Failed prop type: The prop `xs` of `Grid` must be used on `item`.

File: src/pages/CreateProfile/Activities.tsx [hunk 2]
```diff
@@ -0,0 +1,142 @@
+import React from "react";
```
Comment (mayakeeley) [inline src/pages/CreateProfile/Activities.tsx:1]: Should: resolve errors in console

Warning: Each child in a list should have a unique "key" prop.

Comment (mayakeeley) [inline src/pages/CreateProfile/Activities.tsx:1]: Should: ensure when the activity is selected that it is highlighted. Currently when I select an activity, it's only after I select the next activity that the previous one is highlighted

File: src/pages/CreateProfile/Level.tsx [hunk 3]
```diff
@@ -0,0 +1,144 @@
+import React, { useState } from "react";
```
Comment (mayakeeley) [inline src/pages/CreateProfile/Level.tsx:1]: Should: resolve errors in console

Warning: Each child in a list should have a unique "key" prop.

**Review by mayakeeley — CHANGES_REQUESTED** [review]
Looks great Clara! I'm wondering whether there needs to be more validation on each page - i.e. they have to have filled out the goals/bio rather than just having to click on it

Also we need to make note of the fact that the goals suggestions don't currently exist so they will need to be added in at a later date

File: src/pages/CreateProfile/Activities.tsx (hunk 2 above)
Comment (clarafitzgerald) [reply src/pages/CreateProfile/Activities.tsx:1]: This is something I noticed and it’s actually just an issue with having dev tools open, cos the state of hovered on remains. Try it not on mobile version

File: src/pages/CreateProfile/BasicInfo.tsx (hunk 1 above)
Comment (clarafitzgerald) [reply src/pages/CreateProfile/BasicInfo.tsx:1]: I'm not getting this error in the console so not sure how to fix

Comment (clarafitzgerald) [reply src/pages/CreateProfile/BasicInfo.tsx:1]: should be fixed

File: src/pages/CreateProfile/Activities.tsx (hunk 2 above)
Comment (clarafitzgerald) [reply src/pages/CreateProfile/Activities.tsx:1]: should be fixed

File: src/pages/CreateProfile/Level.tsx (hunk 3 above)
Comment (clarafitzgerald) [reply src/pages/CreateProfile/Level.tsx:1]: should be fixed

**Comment by clarafitzgerald** [comment]
> Looks great Clara! I'm wondering whether there needs to be more validation on each page - i.e. they have to have filled out the goals/bio rather than just having to click on it
> 
> Also we need to make note of the fact that the goals suggestions don't currently exist so they will need to be added in at a later date

Added more validation for goals & bio. Are you happy to add a ticket for that "?" on Goals?

**Comment by clarafitzgerald** [comment]
OnClick added to level chip so clicking level or activity navigates to chosen activity

**Review by mayakeeley — APPROVED** [review]

**Pull request merged**